### ISSUE TITLE
"Restore last playback session" is not working. 

### ISSUE THREAD

**Issue by Sadi-Rahman** [anandnet/Harmony-Music#339]
Restores a song from 2 week ago and is stuck to that song, always restores to that one song even when I listen to other songs. I already tried updating and clearing cache.

**Comment by anandnet** [comment]
Yeah, that require little change in logic. Currently if you close app properly using back button after pausing music it will work as expected.

**Comment by Sadi-Rahman** [comment]
So is it a feature? I'll have to close the app properly everytime?

**Comment by anandnet** [comment]
No, absolutely not😀. I mean if you want to use this feature you have to follow this untill new logic will be implemented.

**Comment by guilhermegps** [comment]
thanks bro
